modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -194,7 +201,7 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
   ### loop over time                                                                                 ###
```

Comment:
Looking a bit above, does restart work on remote? Just wanted to check that there weren't points of confusion in the code between the local settings$outdir and the remote outdir.

Reply:
I'm planning to pull out all the `restart` code that are now inside the SDA workflow and make them their own `SDA_restart` function that can basically read the previous output and the new xml and will stay on top of the current SDA function. 
I'll also make sure that I talk about the tags in the documentation.